### ISSUE TITLE
form builder does not handle new select types

### ISSUE THREAD

**Issue by dorey** [kobotoolbox/kpi#922]
- adapt coffeescript code to handle `select_from_list_name` format

**Comment by joshuaberetta** [comment]
@dorey is this related to https://github.com/kobotoolbox/formpack/pull/251#discussion_r657572592?

**Comment by dorey** [comment]
This issue is from around the time that we moved the form builder away from XLSForm syntax. The form builder was originally built to be a sort of "wrapper" around (a csv representation of) an XLSForm. So, the original way we stored select questions and their lists was:

`select_one listname`

We had to support legacy types in javascript/coffeescript because people importing surveys could have chosen any of the aliases, and we would convert them all to `select_one listname`, etc.

Around this time , we switched to parsing aliases and standardizing all the questions in the backend when the form was saved.

So yes, the issues describe the same thing but formpack's standardization code is used when the form is saved into KPI's database , when it's exported, etc. This bug refers to the step of getting it working in the form builder.
